src/ethereum/ethNetwork.js

```diff
@@ -1587,7 +1643,8 @@ export class EthereumNetwork {
     }
     if (blockbookServers.length > 0) {
       blockheight.push(this.checkBlockHeightBlockbook)
-      // txs.push(this.checkTxsBlockbook)
+      tokenBal.push(this.checkAddressBlockbook)
```

Comment:
Delete the tokenBal line. The token balance query is a loop over each enabled currency code which would be lot a redundant fetches for the exact same data. Leaving it in the nonce loop returns the same data without a loop. This is where the original function that grabbed txs should have been the whole time